Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -368,6 +371,30 @@ def download_artifact(url, dir, pr)
     end
   end
 
+  def pr_check_conflicts(name, repo, pr)
+    long_build_pr_files = GitHub.get_long_build_pull_requests_files(name, repo)
+    this_pr_files = GitHub.get_pr_files(name, repo, pr)
+
+    conflicts = []
+    this_pr_files.each do |this_pr_file|
+      long_build_pr_files.each do |long_build_pr_file|
+        if this_pr_file == long_build_pr_file["file"]
+          conflicts.append("#{tap.default_remote}/pull/#{long_build_pr_file["pr"]}")
+        end
+      end
+    end
+    return if conflicts.blank?
+
+    # Raise an error, display the conflicting PR. For example:
+    # Error: You are trying to merge a pull request that conflicts with a long running build in
+    # https://github.com/Homebrew/homebrew-core/pull/10867
+    # https://github.com/Homebrew/homebrew-core/pull/9856
+    odie <<~EOS
+      You are trying to merge a pull request that conflicts with a long running build in
+      #{conflicts * "  \n"}
```

Comment:
```suggestion
        #{conflicts * "  \n"}
```
I think your alignment will come out wrong for multiple PRs without the extra indent.

Reply:
I get this:
```
Error: You are trying to merge a pull request that conflicts with a long running build in
https://github.com/Homebrew/homebrew-core/pull/9000
https://github.com/Homebrew/homebrew-core/pull/105565
```

If I add the extra space in front, I get this, which looks worse:
```
Error: You are trying to merge a pull request that conflicts with a long running build in
  https://github.com/Homebrew/homebrew-core/pull/9000
https://github.com/Homebrew/homebrew-core/pull/105565
```

Reply:
```suggestion
        #{conflicts * "  \n. "}
```
I think this is what you want

Reply:
Oh, sorry, it should actually be
```suggestion
        #{conflicts * "\n  "}
```
Spaces are hard